content/en/news/releases/1.13.x/announcing-1.13.9/index.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Announcing Istio 1.13.9
+linktitle: 1.13.9
+subtitle: Patch Release
+description: Istio 1.13.9 patch release.
+publishdate: 2022-10-07
+release: 1.13.9
+---
+
+This release contains a patch for [CVE-2022-41715](https://github.com/golang/go/issues/55949) and bug fixes to improve robustness. This release note describes what’s different between Istio 1.13.8 and Istio 1.13.9.
```

Comment:
Do we want to link to the actual CVE since we are calling it out?

Reply:
Do you have a recommended link to use here?

Reply:
I don't. Maybe the PSWG does.

Reply:
@jacob-delgado Do you have a recommended link to use here?